features/checkout.feature

```diff
@@ -0,0 +1,16 @@
+Feature: As a buyer
+In order to get printable high resolution image
+I would like to be able to pay for my image.
```

Comment:
Please take a look at the other features in the code and follow the same standard.